React/Views/RCTViewManager.m

```diff
@@ -495,7 +496,38 @@ - (RCTShadowView *)shadowView
     view.validKeysUp = [RCTConvert NSArray:json];
   }
 }
-#endif // ]TODO(macOS ISS#2323203)
+// [TODO(macOS GH#768)
+RCT_CUSTOM_VIEW_PROPERTY(nextKeyViewTag, NSNumber, RCTView)
+{
+    NSNumber *nextKeyViewTag = [RCTConvert NSNumber:json];
+    
+    RCTUIManager *uiManager = [[self bridge] uiManager];
+    NSView *nextKeyView = [uiManager viewForReactTag:nextKeyViewTag];
+     if (nextKeyView) {
+         NSView *targetView = view;
+         // The TextInput component is implemented as a RCTUITextField wrapped by a RCTSinglelineTextInputView,
+         // so we need to get the first subview to properly transfer focus
+         if ([targetView isKindOfClass:[RCTSinglelineTextInputView class]]) {
+             targetView = [[view subviews] firstObject];
+         }
+         if ([nextKeyView isKindOfClass:[RCTSinglelineTextInputView class]]) {
+             nextKeyView = [[nextKeyView subviews] firstObject];
+         }
+         [targetView setNextKeyView:nextKeyView];
+     }
+}
+
+RCT_EXPORT_METHOD(recalculateKeyViewLoop: (nonnull NSNumber *)reactTag)
+{
+    [self.bridge.uiManager addUIBlock:^(RCTUIManager *uiManager, NSDictionary<NSNumber *, RCTUIView *> *viewRegistry) {
+      RCTUIView *view = viewRegistry[reactTag];
+        if (!view) {
+            RCTLogError(@"Cannot find NativeView with tag #%@", reactTag);
+        }
+        [[view window] recalculateKeyViewLoop];
+    }];
+}
```

Comment:
I would like this to instead be a method on UIManager, because it's more a window method than a view method. But that requires me to run Codegen and I don't know how to do that. Perhaps @appden or someone else at Facebook could help?

We also don't need this method for the review, but I think it's a useful addition for that rare client who does elect to completely manually manage their key view loop.